packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -316,63 +316,61 @@ const maxDate = (a: number | string, b: string) => {
-                ) {
-                    // is date
-                    const min = minDate(acc[0], value);
-                    const max = maxDate(acc[1], value);
+    if (!series || series.length === 0) return [];
+
+    let rawValues = [];
+    for (const s of series) {
+        for (const row of rows) {
+            rawValues.push(row[s]?.value.raw);
+        }
+    }
+
+    return rawValues.reduce<(string | number)[]>(
+        (acc, value) => {
+            if (
+                typeof value === 'string' &&
+                dayjs(value, 'YYYY-MM-DD', false).isValid()
+            ) {
+                // is date
+                const min = minDate(acc[0], value);
+                const max = maxDate(acc[1], value);
+
+                return [min, max];
+            } else if (typeof value === 'string' || typeof value === 'number') {
+                // is number or numeric string
+                const currentNumber =
+                    typeof value === 'string' ? parseFloat(value) : value;
+                const currentMin =
+                    typeof acc[0] === 'string' ? parseFloat(acc[0]) : acc[0];
+                const currentMax =
+                    typeof acc[1] === 'string' ? parseFloat(acc[1]) : acc[1];
+
+                if (!isNaN(currentNumber)) {
+                    const min =
+                        currentNumber < currentMin ? currentNumber : currentMin;
+                    const max =
+                        currentNumber > currentMax ? currentNumber : currentMax;
 
                     return [min, max];
-                } else if (
-                    typeof value === 'string' ||
-                    typeof value === 'number'
-                ) {
-                    // is number or numeric string
-                    const currentNumber =
-                        typeof value === 'string' ? parseFloat(value) : value;
-                    const currentMin =
-                        typeof acc[0] === 'string'
-                            ? parseFloat(acc[0])
-                            : acc[0];
-                    const currentMax =
-                        typeof acc[1] === 'string'
-                            ? parseFloat(acc[1])
-                            : acc[1];
-
-                    if (!isNaN(currentNumber)) {
-                        const min =
-                            currentNumber < currentMin
-                                ? currentNumber
-                                : currentMin;
-                        const max =
-                            currentNumber > currentMax
-                                ? currentNumber
-                                : currentMax;
-
-                        return [min, max];
-                    }
                 }
-                return acc;
-            },
-            [0, 0],
-        );
+            } else {
+                // TODO: this case comes up more than it should given that
```

Comment:
should we add a log here instead of silently fail ?

Reply:
Would you log to the console? I wasn't entirely sure what it would say. You think it should log to the console? I put this here cause I figure it's helpful for devs, but I didn't throw or anything because there is nothing a user can do about it.

Reply:
I'm going to merge without this. But I might look into this TODO. If you want, I'm happy to add a log here.